cmw-dto/src/main/java/com/liberologico/cloudesire/cmw/model/enums/EventRecipientType.java

```diff
@@ -1,5 +1,8 @@
 package com.liberologico.cloudesire.cmw.model.enums;
 
+import io.swagger.annotations.ApiModel;
+
+@ApiModel( "If the recipient of the event is a company or a platform endpoint" )
 public enum EventRecipientType
 {
     COMPANY, PLATFORM
```

Comment:
COMPANY - "The event recipient is an endpoint defined by the vendor"
PLATFORM - "The event recipient is a platform endpoint"